packages/leva/src/components/Leva/Filter.tsx

```diff
@@ -74,30 +92,37 @@ export function TitleWithFilter({ setFilter, onDrag, toggle, toggled }: TitleWit
 
   return (
     <>
-      <StyledTitleWithFilter>
+      <StyledTitleWithFilter mode={drag ? 'drag' : undefined}>
         <Icon active={!toggled} onClick={() => toggle()}>
           <Chevron toggled={toggled} width={12} height={8} />
         </Icon>
-        <Drag {...bind()}>
-          <svg width="20" height="10" viewBox="0 0 28 14" xmlns="http://www.w3.org/2000/svg">
-            <circle cx="2" cy="2" r="2" />
-            <circle cx="14" cy="2" r="2" />
-            <circle cx="26" cy="2" r="2" />
-            <circle cx="2" cy="12" r="2" />
-            <circle cx="14" cy="12" r="2" />
-            <circle cx="26" cy="12" r="2" />
-          </svg>
-        </Drag>
-        <Icon active={filterShown} onClick={() => setShowFilter((f) => !f)}>
-          <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 0 20 20">
-            <path d="M9 9a2 2 0 114 0 2 2 0 01-4 0z" />
-            <path
-              fillRule="evenodd"
-              d="M10 18a8 8 0 100-16 8 8 0 000 16zm1-13a4 4 0 00-3.446 6.032l-2.261 2.26a1 1 0 101.414 1.415l2.261-2.261A4 4 0 1011 5z"
-              clipRule="evenodd"
-            />
-          </svg>
-        </Icon>
+        <TitleContainer {...(drag ? bind() : {})} mode={drag ? 'drag' : undefined}>
+          {title === undefined && drag ? (
+            <svg width="20" height="10" viewBox="0 0 28 14" xmlns="http://www.w3.org/2000/svg">
+              <circle cx="2" cy="2" r="2" />
+              <circle cx="14" cy="2" r="2" />
+              <circle cx="26" cy="2" r="2" />
+              <circle cx="2" cy="12" r="2" />
+              <circle cx="14" cy="12" r="2" />
+              <circle cx="26" cy="12" r="2" />
+            </svg>
+          ) : (
+            title
+          )}
+          {!filterEnabled && <DisabledIconPlaceholder />}
```

Comment:
I added this so the dots are still in the center of the title bar in case filtering is disabled.

Reply:
This could also be applied via padding on the TitleContainer. Let me know which method you prefer

Reply:
I think paddingRight would be better. You can actually create a variant for the TitleContainer class?

Reply:
See https://github.com/pmndrs/leva/pull/185/commits/529c3d36d9c33463138432672e42b187f4f9df8a